src/main/java/com/sampleweb/pages/DVDPage.java

```diff
@@ -0,0 +1,63 @@
+package com.sampleweb.pages;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@SuppressWarnings("unused")
+public class DVDPage extends HomePage {
+
+    public static String TITLE = "DVD & Blu-ray";
+
+    @FindBy(css = ".bxw-pageheader__title__text>h1")
+    public WebElement title;
+
+    @FindBy(css = "div#nav-subnav>a:first-child>span")
+    public WebElement pageLink;
+
+    @FindBy(xpath = "//*[@id='leftNav']//i[contains(@class, 'a-icon-prime')]")
+    public WebElement primeCheckbox;
+
+    @FindBy(xpath = "//*[@class='s-result-list s-search-results sg-row']/div")
+    public List<WebElement> products;
+
+    @FindBy(xpath = "//span[text()='4 Stars & Up']")
+    public WebElement fourStars;
+
+    @FindBy(xpath = "//*[@id=\"s-result-sort-select\"]/option[text()='Avg. Customer Review']")
```

Comment:
The `*`could be `select`. Also you inconsistently have used `\"` here instead of `'`.